Writerside/topics/pplib-Build-an-Auto.md

```diff
@@ -471,4 +474,29 @@ public class RobotContainer {
 }
 ```
 
-</snippet>
\ No newline at end of file
+</tab>
+<tab title="Python" group-key="python">
+
+```Python
+from pathplannerlib.auto import AutoBuilder
+
+class RobotContainer:
+    
+    def __init__():
+
+        # Build an auto chooser. This will use Commands.none() as the default option.
+        self.autoChooser = AutoBuilder.buildAutoChooser()
+        
+        # Another option that allows you to specify the default auto by its name
+        # self.autoChooser = AutoBuilder.buildAutoChooser("My Default Auto")
+        
+        SmartDashboard.putData("Auto Chooser", self.autoChooser)
+    
+    def getAutonomousCommand():
+        return self.autoChooser.getSelected()
+```
+
+</tab>
+</tabs>
+
+</snippet>
```

Comment:
I believe this is a stray closing tag
```suggestion

```

Reply:
Is it? That was there before, I simply moved it down. The md originally opened with <snippet id="build-an-auto"> and ended with </snippet> immediately after the java example code.

Reply:
Yeah you're right. Forgot that file is different